server/monitor.go

```diff
@@ -2260,6 +2260,13 @@ func (s *Server) accountInfo(accName string) (*AccountInfo, error) {
 			vrIssues[i] = ExtVrIssues{v.Description, v.Blocking, v.TimeCheck}
 		}
 	}
+	collectRevocations := func(revocations map[string]int64) map[string]time.Time {
+		rev := map[string]time.Time{}
```

Comment:
That will likely cause that collectRevocations will always return a non-nil map, which then means that `ExtExport.RevokedAct` will never be nil (and so omitempty does not help). Should you declare as `var rev map[string]time.Time` and create only in the for-loop if nil?

Reply:
fixed